src/test/java/net/snowflake/client/jdbc/ResultSetStructuredTypesLatestIT.java

```diff
@@ -413,5 +437,16 @@ private void withFirstRow(String sqlText, ThrowingConsumer<ResultSet, SQLExcepti
       consumer.accept(rs);
     }
   }
-  ;
+
+  enum ResultSetFormatType {
```

Comment:
Sme here - @SnowflakeJdbcInternalApi?

Reply:
Do we need this annotation in tests? It is not visible for clients.

Reply:
@sfc-gh-pfus is right, removed it.